docs/5.0/preview/cloud.md

```diff
@@ -100,8 +104,9 @@ $ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
 
 ### When will a security audit be available?
 
-The Teleport Cloud security audit starts in January.
-We will publish it in Q1 2021 after fixing any security issues found.
+The Teleport Cloud security audit starts in January. 
+We will work on fixing any security issues that are identified and 
+publish it in Q1 2021.
 
 ### What does Teleport Cloud run in?
```

Comment:
This didn't make sense to me. But if it does to you that is fine. I would think that we use Gravity to deploy and it runs in K8s?